voidcrew/code/game/objects/items/devices/walkman/cassette.dm

```diff
@@ -0,0 +1,166 @@
+/obj/item/device/cassette_tape
```

Comment:
OK so heres the big one. These should all be datumised. At least, all the specific tapes. Keep this generic tape, and then make some datums for each casset type, and use that to apply all the vars like songs, etc.

Reply:
you can take a look at my overmap refactor to get an idea at what i mean

Reply:
think i did this now